doc/explanation/components/reactive_html_components.md

```diff
@@ -0,0 +1,781 @@
+# Building `ReactiveHTML` Components
```

Comment:
The section headers are too long. Either they should be shortened or the width of the sidebar should be changed.

![image](https://github.com/holoviz/panel/assets/42288570/97ebc2cd-7034-43bd-ad12-a42957b26c9d)